server/sttp-stub-server/src/main/scala/sttp/tapir/server/stub/SttpRequestBody.scala

```diff
@@ -45,8 +56,7 @@ class SttpRequestBody[F[_]](implicit ME: MonadError[F]) extends RequestBody[F, A
     case InputStreamBody(b, _)      => Left(toByteArray(b))
     case FileBody(f, _)             => Left(f.readAsByteArray)
     case StreamBody(s)              => Right(s)
-    case MultipartBody(_) =>
-      throw new IllegalArgumentException("Stub cannot handle multipart bodies")
+    case MultipartBody(parts)       => Right(parts)
```

Comment:
this confuses multiparts & streams. We should probably just have a special-case in `toStream`, and use this body only for "normal" requests